airflow/providers/google/__init__.py

```diff
@@ -14,27 +15,24 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+#
+# NOTE! THIS FILE IS AUTOMATICALLY GENERATED AND WILL BE
+# OVERWRITTEN WHEN PREPARING DOCUMENTATION FOR THE PACKAGES.
+#
+# IF YOU WANT TO MODIFY IT, YOU SHOULD MODIFY THE TEMPLATE
+# `PROVIDER__INIT__PY_TEMPLATE.py.jinja2` IN the `dev/provider_packages` DIRECTORY
+#
 from __future__ import annotations
 
-import importlib
-import logging
+import packaging.version
 
-# HACK:
-# Sphinx-autoapi doesn't like imports to excluded packages in the main module.
-conf = importlib.import_module("airflow.configuration").conf  # type: ignore[attr-defined]
+import airflow
 
-PROVIDERS_GOOGLE_VERBOSE_LOGGING: bool = conf.getboolean(
```

Comment:
I believe this (and corresponding `confg_templates` directory has been used internally by Poilidea team to increase logging for providers, but it has never ever been exposed or communicated to anyone (I will double check it).

Reply:
I **think** the idea was to those config_templates from providers could be eventually merged into the main configuration templates (bad idea) and it has never been actually implemented.

Reply:
@mik-laj - does it ring a bell?